open-company/open-company-web
# Disable fullstory in local development and on staging.

### Review with https://github.com/belucid/open-company-infrastructure/pull/67

To test:
- run local dev
- [x] check that `init_fullstory` is not called
- [x] check that no call is done to fullstory
- deploy to staging
- [x] check that `init_fullstory` is not called
- [x] check that no call is done to fullstory
- merge
- deploy to beta
- [ ] check that `init_fullstory` IS called
- [ ] check that we are sending fullstory the bundles
